modules/navier_stokes/tests/ins/open_bc/RZ_do_nothing_bcs_cone.i

```diff
@@ -0,0 +1,140 @@
+[GlobalParams]
+  rho = 1
+  mu = 1
+  integrate_p_by_parts = false
+  gravity = '0 0 0'
+  coord_type = RZ
+[]
+
+[Mesh]
+  file = '2d_cone.msh'
+[]
+
+[Problem]
+[]
+
+[Preconditioning]
+  [./SMP_PJFNK]
+    type = SMP
+    full = true
+    solve_type = Newton
+  [../]
+[]
+
+[Executioner]
+  type = Steady
+  petsc_options_iname = '-pc_type -pc_factor_shift_type -pc_factor_shift_amount -ksp_type'
+  petsc_options_value = 'lu NONZERO 1.e-10 preonly'
```

Comment:
I'd rather not check in a test which uses `-pc_type lu`.  This is fine for testing, but it means this test would not be able to run in parallel...